docs/user-guide/api.md

```diff
@@ -52,11 +52,14 @@ For more information and examples, check out our [API documentation](https://api
 
 ## Authorization and Authentication
```

Comment:
Title should be in the order that you're talking about the topics. In this case, it should be "Authentication and Authorization"